english/lisp/lisp.md

```diff
@@ -0,0 +1 @@
+The LISP interpreter starts with a flash of blue-green flame!
```

Comment:
i've fixed the path bug and it works on my local fork.
i think i've updated my commit so it updates the pull request.

cbuckey-uda made a similar suggestion  about changing the dir structure.
i'll try that as a later pull request.

thanks,
amaresh